src/logger/UsageLoggers.go

```diff
@@ -0,0 +1,78 @@
+package logger
+
+import (
+	"sync"
+	//Library used for getting environment variables and other useful env things
+	"os"
+	//used for converting the string returned by lookupEnv
+	"strconv"
+)
+
+//name change since helper uses the name "once"
+var onceUsageLoggers sync.Once
+
+type UsageLoggers struct {
+	bricked  bool
+	disabled bool
+}
+
+var usageLoggers *UsageLoggers
+
+func GetUsageLoggers() *UsageLoggers {
+	onceUsageLoggers.Do(func() {
+		//TODO: impement functionality of "true".equals(System.getenv("USAGE_LOGGERS_DISABLE"))"
+		//in java bricked is final and both are static
```

Comment:
So within go the static nature of these variables will partially be covered by the singleton construction with `.Do` and the final nature will be accomplished via the lowercase naming which will keep these contained within the package (meaning they will remain unchanged unless manipulated inside the package).